softlearning/samplers/simple_sampler.py

```diff
@@ -19,6 +18,12 @@ def __init__(self, **kwargs):
         self._n_episodes = 0
         self._current_observation = None
         self._total_samples = 0
+        ################################################################################################################
+        self.z_dim = z_dim                                                                                              # dimension of latent space
+        self.z_type = z_type
+        self.z_type = 'normal'                                                                                          # type of latent distribution
```

Comment:
it is a little dangerous to offer the param but then silently overwrite it. better either remove the param completely and just hardcode gaussian or raise a NotImplementedError if the value of z_type is not 'normal'